Beam/resources/assets/js/remplib.js

```diff
@@ -48,6 +48,16 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
 
         timeSpentActive: false,
 
+        trackedProgress: [],
+
+        trackedArticle: null,
+
+        trackedProgressInterval: 5,
+
+        trackedProgressPath: this.config.tracker.readingProgress.target,
+
+        dt: 0,
```

Comment:
Let's rename `dt` to something more explanatory. I have to read the code to get the idea what's that for.